core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -774,10 +774,8 @@ protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
             analysis.setUpdateType("ANALYZE");
             analysis.setUpdateTarget(tableName, Optional.empty(), Optional.empty());
 
-            // verify the target table exists and it's not a view
```

Comment:
This isn't needed because views do not have table handles, so it will fail naturally.